igvm/vm.py

```diff
@@ -119,6 +119,18 @@ def run(self, command, silent=False, with_sudo=True):
             else:
                 return super(VM, self).run(command, silent=silent)
 
+    def read_file(self, path):
+        """ Read a file from a running VM or a mounted image on HV
+        """
```

Comment:
No space after `"""` and closing `"""` to the same line please.  See https://www.python.org/dev/peps/pep-0257/#one-line-docstrings

Reply:
Fixed.